helpers/models/Lesson.ts

```diff
@@ -0,0 +1,25 @@
+import { DataTypes, Model } from 'sequelize'
+
+export class Lesson extends Model {
+  public id!: number
+  public description!: string
+  public docUrl?: string
+  public githubUrl?: string
+  public videoUrl?: string
+  public order!: number
+  public titel!: string
```

Comment:
typo?

Reply:
GOOD CATCH @michaelbayday 

How'd you grow such sharp eyes?